runtime/doc/todo.txt

```diff
@@ -2598,9 +2598,6 @@ Win32 GUI known bugs:
 8   When the 'shell' cannot be executed, should give an appropriate error msg.
     Esp. for a filter command, currently it only complains the file could not
     be read.
-7   Add an option to add one pixel column to the character width?  Lucida
-    Console italic is wider than the normal font ("d" overlaps with next char).
-    Opposite of 'linespace': 'columnspace'.
```

Comment:
This file (todo.txt) should not be modified at all.  If upstream accepts the patch, then Bram will do it.  Otherwise it should be left alone.